index.js

```diff
@@ -144,6 +145,31 @@ sassLint.lintText = function (file, options, configPath) {
 };
 
 /**
+ * Handles ignored files for plugins such as the gulp plugin. Checks every file passed to it against
+ * the ignores as specified in our users config or passed in options.
+ *
+ * @param {object} file - The file/text to be linted
+ * @param {object} options - The user defined options directly passed in
+ * @param {object} configPath - Path to a config file
+ * @returns {object} Return the results of lintText - a results object
+ */
+sassLint.lintFileText = function (file, options, configPath) {
+  var config = this.getConfig(options, configPath);
+  var ignores = config.files ? config.files.ignore : [];
```

Comment:
var config = this.getConfig(options, configPath),
    ignores = config.files ? config.files.ignore : [];